baselines/dom.generated.d.ts

```diff
@@ -3863,18 +3863,57 @@ interface DhKeyGenParams extends Algorithm {
 }
 
 interface DocumentEventMap extends GlobalEventHandlersEventMap {
+    "abort": Event;
+    "abort": ProgressEvent;
     "abort": UIEvent;
-    "activate": Event;
-    "beforeactivate": Event;
-    "beforedeactivate": Event;
+    "afterprint": Event;
+    "animationend": AnimationEvent;
+    "animationiteration": AnimationEvent;
+    "animationstart": AnimationEvent;
+    "appinstalled": Event;
+    "audioend": Event;
+    "audioprocess": AudioProcessingEvent;
+    "audiostart": Event;
+    "beforeprint": Event;
+    "beforeunload": BeforeUnloadEvent;
+    "blocked": ;
     "blur": FocusEvent;
+    "boundary": SpeechSynthesisEvent;
+    "cached": Event;
     "canplay": Event;
     "canplaythrough": Event;
     "change": Event;
+    "checking": Event;
     "click": MouseEvent;
-    "contextmenu": PointerEvent;
+    "close": Event;
+    "complete": ;
+    "complete": OfflineAudioCompletionEvent;
+    "compositionend": CompositionEvent;
+    "compositionstart": CompositionEvent;
+    "compositionupdate": CompositionEvent;
+    "contextmenu": MouseEvent;
+    "copy": ClipboardEvent;
+    "cut": ClipboardEvent;
     "dblclick": MouseEvent;
-    "deactivate": Event;
+    "devicechange": Event;
+    "devicelight": DeviceLightEvent;
+    "devicemotion": DeviceMotionEvent;
+    "deviceorientation": DeviceOrientationEvent;
+    "deviceproximity": DeviceProximityEvent;
+    "DOMActivate": UIEvent;
+    "DOMAttributeNameChanged": MutationNameEvent;
```

Comment:
I think the current DOM spec does not include these mutation events.

Reply:
Thanks for reply

TBH I'm not entirely sure where to reference the latest spec' these days..?

Specifically, which events are you referring to or all of them in the PR?

Cheers

Reply:
Check out this one https://dom.spec.whatwg.org/#goals

>By defining a replacement for the "Mutation Events" and "Mutation Name Event Types" chapters of DOM Level 3 Events [uievents-20031107] as the old model was problematic.